src/components/organisms/Top.tsx

```diff
@@ -2,24 +2,32 @@ import React, { useState, useEffect } from "react";
 import styles from "@/styles/components/organisms/Top.module.scss";
 import Weather from "@/components/molecules/Weather";
 import Livedata from "./Livedata";
+import axios from "axios";
 
 export default function Top() {
-  const [lat, setLat] = useState<number | null>();
-  const [long, setLong] = useState<number | null>();
   const [currTemp, setCurrTemp] = useState<number | null>();
   const [currHumid, setCurrHumid] = useState<number | null>();
+  const [deviceId, setDeviceId] = useState<number | null>(1);
+  const [data, setData] = useState<any>([]);
 
```

Comment:
It is hard that what type of `data` has from the name and `any` type.
Add a specific type and change the variable name.(Like weatherData?)